pom.xml

```diff
@@ -130,46 +97,23 @@
         </executions>
       </plugin>
 
-
       <plugin>
         <groupId>com.mycila</groupId>
         <artifactId>license-maven-plugin</artifactId>
-        <configuration>
-          <header>etc/license.txt</header>
+        <configuration combine.self="override">
+          <header>${rootdir}/etc/license.txt</header>
           <strictCheck>true</strictCheck>
-          <aggregate>true</aggregate>
           <excludes>
+            <exclude>README.md</exclude>
             <exclude>jitpack.yml</exclude>
             <exclude>LICENSE.txt</exclude>
             <exclude>install-jdk.sh</exclude>
+            <exclude>mvnw</exclude>
+            <exclude>mvnw.cmd</exclude>
+            <exclude>.mvn/wrapper/maven-wrapper.properties</exclude>
+            <exclude>.mvn/wrapper/MavenWrapperDownloader.java</exclude>
           </excludes>
         </configuration>
-        <executions>
-          <execution>
-            <goals>
-              <goal>check</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>enforce-java</id>
-            <goals>
-              <goal>enforce</goal>
-            </goals>
-            <configuration>
-              <rules>
-                <requireJavaVersion>
-                  <version>[14.0,)</version>
-                </requireJavaVersion>
-              </rules>
-            </configuration>
-          </execution>
-        </executions>
       </plugin>
```

Comment:
Did that go elsewhere?

Reply:
Yes, the parent performs the check using the `${java.version}` property setup in this POM